php/commands/media.php

```diff
@@ -18,6 +18,9 @@ class Media_Command extends WP_CLI_Command {
 	 * [--skip-delete]
 	 * : Skip deletion of the original thumbnails. If your thumbnails are linked from sources outside your control, it's likely best to leave them around. Defaults to false.
 	 *
+	 * [--only-missing]
+	 * : Regenerate only media files that have some generated size missing. No effect if used without --skip-delete.
```

Comment:
Wouldn't it be better if using `--only-missing` would automatically set the `--skip-delete` flag?